### ISSUE TITLE
Allow not to update training losses in any step

### ISSUE THREAD

**Issue by Libric0** [Libric0/loss_watch#2]
Right now, the update function expects a training loss. 

This seems useful for small-scale models, but as soon as you go into millions of epochs, it may clutter your ram, depending on your machine.

The alternative would be to completely remove the train losses and see them as just another validation loss, but keep train_step as an optional parameter, not as a kwarg, in the update function. So it is still something special and still drawn on top.
